LocalPackages/SystemExtensionManager/Sources/SystemExtensionManager/SystemExtensionManager.swift

```diff
@@ -47,21 +47,9 @@ public struct SystemExtensionManager {
     /// - Returns: The system extension version when it's updated, otherwise `nil`.
     ///
     public func activate(waitingForUserApproval: @escaping () -> Void) async throws -> String? {
-        /// Documenting a workaround for the issue discussed in https://app.asana.com/0/0/1205275221447702/f
-        ///     Background: For a lot of users, the system won't show the system-extension-blocked alert if there's a previous request
-        ///         to activate the extension.  You can see active requests in your console using command `systemextensionsctl list`.
-        ///
-        ///     Proposed workaround: Just open system settings into the right section when we detect a previous activation request already exists.
-        ///
-        ///     Tradeoffs: Unfortunately we don't know if the previous request was sent out by the currently runing-instance of this App
-        ///         or if an activation request was made, and then the App was reopened.
-        ///         This means we don't know if we'll be notified when the previous activation request completes or fails.  Because we
-        ///         need to update our UI once the extension is allowed, we can't avoid sending a new activation request every time.
-        ///         For the users that don't see the alert come up more than once this should be invisible.  For users (like myself) that
-        ///         see the alert every single time, they'll see both the alert and system settings being opened automatically.
-        ///
-        if hasPendingActivationRequests() {
-            openSystemSettingsSecurity()
```

Comment:
The diff seems a bit complicated but it's really not.  I moved these comments and code into a dedicated method that's only run before macOS 15.  We no longer want to be trying to do what we're doing here in Sequoia.

For context this workaround is to try and open the right settings location for the user ourselves.  This is not great and can actually be really confusing when broken (like now).  Starting on Sequoia this is no longer done.